doc/src/tools/kani.md

```diff
@@ -84,21 +84,24 @@ pub mod verify {
 }
 ```
 
-### Step 2
+### Step 2 - Run the Kani verify-std subcommand
 
-Run the following command in your local terminal:
+To aid the Rust Standard Library verification effort, Kani provides a sub-command out of the box to help you get started.
+Run the following command in your local terminal (Replace "path/to/library" and "path/to/target" with your local paths):
 
-`kani verify-std -Z unstable-options "path/to/library" --target-dir "/path/to/target" -Z function-contracts -Z stubbing`.
+```
+kani verify-std -Z unstable-options "path/to/library" --target-dir "/path/to/target" -Z function-contracts -Z mem-predicates
```

Comment:
I would just tell them to use `/tmp/verify-rust-std` as their target dir; does it matter what they provide? My understanding was that this is just a temporary folder for Kani to use, so there doesn't need to be some existing local path that people need to provide (like for library).

Reply:
Oh I see you gave an example below -- never mind, this is fine.